index.html

```diff
@@ -1455,13 +1455,23 @@
 
 ## Sensitive Information {#hl-sensitive-information}
 
-Contributes to [=correlation=], [=identification=], [=secondary use=], and
-[=disclosure=].
+<div class="practice">
+<p>
+  <span class="practicelab" id="principle-vulnerability">
+    System designers
+    should not assume that particular information is or is not sensitive.
+    Whether information is considered sensitive can vary depending on a
+    [=person=]'s circumstances, the [=context=] of an interaction, and can
```

Comment:
This isn't parallel. Suggest:

```suggestion
    [=person=]'s circumstances and the [=context=] of an interaction, and it can
```